tests/integration/hive_metastore/test_tables.py

```diff
@@ -52,3 +52,36 @@ def test_describe_all_tables_in_databases(ws, sql_backend, inventory_schema, mak
     assert all_tables[view.full_name].object_type == "VIEW"
     assert all_tables[view.full_name].view_text == "SELECT 2+2 AS four"
     assert all_tables[view.full_name].what == What.VIEW
+
+
+@retried(on=[NotFound, TimeoutError], timeout=timedelta(minutes=5))
```

Comment:
```suggestion
@retried(on=[NotFound], timeout=timedelta(minutes=2))
```
smaller timeouts and tighter criteria

Reply:
Done